src/sentry/utils/snuba.py

```diff
@@ -1070,12 +1070,13 @@ def _raw_mql_query(
     # Enter hub such that http spans are properly nested
     with thread_hub, timer("mql_query"):
         referrer = headers.get("referer", "unknown")
+        serialized_req = request.serialize_mql()
         with thread_hub.start_span(op="snuba_mql.validation", description=referrer) as span:
             span.set_tag("snuba.referrer", referrer)
             # TODO: This can be changed back to just `serialize` after we remove SnQL support for MetricsQuery
-            body = request.serialize_mql()
+            body = serialized_req
```

Comment:
is the TODO above still relevant?

Reply:
good catch, I moved the comment to the relevant line